manifest_staging/deployment/provider-azure-installer.yaml

```diff
@@ -29,19 +29,17 @@ spec:
           args:
             - --endpoint=unix:///provider/azure.sock
             - --construct-pem-chain=false
-          lifecycle:
-            preStop:
-              exec:
-                command:
-                  - "rm /provider/azure.sock"
-          readinessProbe:
```

Comment:
do we want to remove `readinessProbe`?

Reply:
`readinessProbes` are typically used by kubelet to determine if the pod can receive traffic. That really doesn't apply for the driver-provider communication. As long as the socket exists, the driver will make a RPC call to the provider.